src/route-utils.js

```diff
@@ -138,6 +138,21 @@ export function parseRoute({params, routes, location}) {
 class FilterState {
   constructor(state) {
     this.state = state;
+    if (this.state.repoInfos.length === 0) {
+      const REPOSITORIES = process.env['REPOSITORIES'];
+      const repositories = REPOSITORIES || 'coala';
```

Comment:
Instead of hardcoding 'coala' here, I think it's more appropriate to have it on webpack config. That way we don't have to worry about 'coala' if we use this env in another module.

Reply:
Or maybe just remove 'coala' ? @jayvdb

Reply:
Yes, definitely remove it.

If `REPOSITORIES` is empty (such as for pull requests on travis), the dashboard should be shown as before.

A slightly easier way of solving this is https://github.com/coala/gh-board/issues/3#issuecomment-386508826